src/frontends/pytorch/src/utils.cpp

```diff
@@ -555,6 +555,25 @@ Output<Node> masked_fill(ov::pass::NodeRegistry& rg,
     return rg.make<opset10::Select>(bool_mask, _value, data);
 }
 
+Output<Node> concat_dims_to_shape(const NodeContext& context, size_t begin, size_t end) {
+    OutputVector list_elems;
+    for (size_t i = begin; i < end; i++) {
+        if (context.get_input_type(i).as<type::List>().element_type.is<type::PyScalar>()) {
+            auto const_val = context.const_input<int32_t>(i);
+            std::vector<int32_t> dim_vec;
+            dim_vec.push_back(const_val);
+            auto dim_const = ov::op::v0::Constant::create(element::i32, Shape{1}, dim_vec);
+            list_elems.push_back(dim_const);
+        } else {
+            auto converted_dim = context.mark_node(
+                std::make_shared<ov::op::v0::Convert>(context.get_input(static_cast<int>(i)), element::i32));
```

Comment:
Do you really need to convert inputs?
Also, why don't you unsqueeze inputs? If it was an element of list it should be unsqueezed, shouldn't it?

Reply:
Removed the convert inputs. Unsqueeze inputs should work fine as long as we are sure that the input is scalar. I will test and push if it doesn't break the execution of any of the targeted models.